dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/generator/GenerateTagFileVisitor.java

```diff
@@ -366,6 +366,8 @@ protected void generateInitSection(ValidateTagFileResult validatorResult) throws
             GeneratorUtils.generate_tagCleanUp_methods(writer, !jspOptions.isDisableResourceInjection()); // PH49514
         }
 
+        GeneratorUtils.generate_finalCleanUp_method(writer, jspOptions);
```

Comment:
Ah, I see what you mean.  I created a new method, and it needs to be in the generated tag classes, too. Otherwise, the code encounters a method not found exception. 

Before, any clean up with done directly in the final method. I can get an example, if you'd like.

Reply:
Remember, this generate the method in the writer (that's passed in), and doesn't actually clean anything in this invocation on line 369. 

Hope that makes sense.